upgrades/3.x/3.10.11/README.adoc

```diff
@@ -19,3 +18,13 @@ As of `3.10.11` the following rules apply:
 - Platform response policies are executed *after* API response policies
 
 
+== Liquibase execution
+If you use a JDBC repository, we recommend you disabling the liquibase scripts execution by the gateway. It will avoid some conflicts when upgrading and starting Gateway and Management API in the same time.
```

Comment:
```suggestion
If you use a JDBC repository, we recommend you disable the liquibase scripts execution by the gateway. It will avoid some conflicts when upgrading and starting Gateway and Management API at the same time.
```